File: Backend/ServerBackend/src/main/java/com/fran/dev/potjera/serverbackend/models/gamesession/GameSessionPlayer.java
package com.fran.dev.potjera.serverbackend.models.gamesession;

import lombok.Builder;
import lombok.Data;

@Builder
@Data
public class GameSessionPlayer {
    Long playerId;
    String playerName;
    Float moneyWon;
    Boolean isEliminated;
    Boolean isCaptain;
    Boolean isHunter;
    Boolean isHost;
    Boolean hasPlayedBoard;
}

File: AndroidApp/app/src/main/java/com/fran/dev/potjera/android/app/game/models/GameSessionPlayer.kt
package com.fran.dev.potjera.android.app.game.models

data class GameSessionPlayer(
    val playerId: Long,
    val playerName: String,
    val moneyWon: Float,
    val isEliminated: Boolean,
    val isCaptain: Boolean,
    val isHunter: Boolean,
    val isHost: Boolean,
    val hasPlayedBoard: Boolean
)